### PR TITLE
Chapter16 regions

### PR DESCRIPTION
**Pull request by HannaMichaelis**
## Summary

Contributes to Intellitect/EssentialCSharpManuscript#116

### CODE REVIEW

**Review by Keboo — APPROVED** [review]

File: src/Chapter16/Listing16.11.SelectingTupleFollowingTheGroupbyClause.cs [hunk 1]
```diff
@@ -1,45 +1,54 @@
...
+            IEnumerable<
+                (bool IsContextualKeyword, IGrouping<bool, string> Items)> 
+                    selection =
                 from groups in keywordGroups
                 select
                 (
                     IsContextualKeyword: groups.Key,
                     Items: groups
                 );
+            #endregion HIGHLIGHT
```
Comment (BenjaminMichaelis) [inline src/Chapter16/Listing16.11.SelectingTupleFollowingTheGroupbyClause.cs:32]:
```suggestion
```
nit: extra line

File: src/Chapter16/Listing16.11.SelectingTupleFollowingTheGroupbyClause.cs [hunk 2]
```diff
@@ -1,45 +1,54 @@
...
         {
+            #region HIGHLIGHT
             IEnumerable<IGrouping<bool, string>> keywordGroups =
                 from word in CSharp.Keywords
                 group word by word.Contains('*');
 
-            IEnumerable<(bool IsContextualKeyword, IGrouping<bool, string> Items)> selection =
+            IEnumerable<
+                (bool IsContextualKeyword, IGrouping<bool, string> Items)> 
+                    selection =
```
Comment (BenjaminMichaelis) [inline src/Chapter16/Listing16.11.SelectingTupleFollowingTheGroupbyClause.cs]:
```suggestion
            IEnumerable<(bool IsContextualKeyword, 
                IGrouping<bool, string> Items)> selection =
```
I think this is a better split point

File: src/Chapter16/Listing16.12.SelectingWithoutTheQueryContinuation.cs [hunk 3]
```diff
@@ -1,29 +1,34 @@
...
             GroupKeywords1();
         }
-
+        #endregion EXCLUDE
         private static void GroupKeywords1()
         {
-            IEnumerable<(bool IsContextualKeyword, IGrouping<bool, string> Items)> selection =
+            IEnumerable<
+                (bool IsContextualKeyword, IGrouping<bool, string> Items)> 
+                    selection =
```
Comment (BenjaminMichaelis) [inline src/Chapter16/Listing16.12.SelectingWithoutTheQueryContinuation.cs]:
```suggestion
            IEnumerable<(bool IsContextualKeyword, 
                IGrouping<bool, string> Items)> selection =
```
I think this is a better split point

**Review by BenjaminMichaelis — APPROVED** [review]
Looks good! Thanks @HannaMichaelis!

**Pull request merged**